libs/community/langchain_community/vectorstores/chroma.py

```diff
@@ -41,11 +41,17 @@ def _results_to_docs_and_scores(results: Any) -> List[Tuple[Document, float]]:
     return [
         # TODO: Chroma can do batch querying,
         # we shouldn't hard code to the 1st result
-        (Document(page_content=result[0], metadata=result[1] or {}), result[2])
+        (
+            Document(
+                page_content=result[0], metadata=(result[1] | {"id": result[3]}) or {}
```

Comment:
this looks like if you have metadata it will override an `id` prop... can we think of a different way to handle this? i think it may really confuse some users, albeit an edge case